en/micro-integrator/docs/references/connectors/amazonlambda-connector/amazonlambda-connector-example.md

```diff
@@ -0,0 +1,262 @@
+3. Micro Integrator will be started and the composite application will be deployed. You can further refer to the application deployed through the CLI tool. Make sure you first export the PATH as below.
+
+    ```
+    $ export PATH=/path/to/mi/cli/directory/bin:$PATH\
+    ```
+{! /references/connectors/exporting-artifacts.md !}
+
+## Create Amazon Lambda Deployment Package (Lambda function) 
+In this scenario we created sample AWS Deployment Package (Lambda function) in Python.
+
+1. Our sample Deployment Package would look similar to the following (source view : addingNumbers.py).
+```
+import json
+   
+   print('Loading function')
+   
+   def addingNumbers(event, context):
+       #print("Received event: " + json.dumps(event, indent=2))
+       value1 = event['key1']
+       value2 = event['key2']
+       print("value1 = " + value1)
+       print("value2 = " + value2)
+       return float(value1) + float(value2) # Echo back the addition of two keys
+       #raise Exception('Something went wrong')
+```
+
+2. Create a ZIP archive.
+
+Please use command line terminal or shell to run following commands.Commands are shown in listings preceded by a prompt symbol ($) and the name of the current directory, when appropriate:
+```
+~/Documents$ zip addingNumbers.zip addingNumbers.py
+ adding: addingNumbers.py.py (deflated 17%)
+```
+## Upload Amazon Lambda Deployment Package (ZIP archive) in to the AWS S3 bucket
+1. Log in to the AWS Management Console.
+2. Navigate in to the created S3 bucket (Ex: eiconnectortest).
+3. Click Upload button.
+4. Select created Amazon Lambda Deployment Package (ZIP archive) and Upload.
+
+## Create Execution Role
+1. You need to create an Execution Role using [Amazon Lambda Connector Configuration](../amazonlambda-configuration.md) documentation. 
+
+## Deployment
+Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
+
+{!references/connectors/deploy-capp.md!}
+
+## Testing
+1. Login in to the Micro Integrator CLI tool.
+
+    ```
+    ./mi remote login
+    ```
+2. Provide default credentials admin for both username and password.
+
+3. In order to view the proxy services deployed, execute the following command.
+
+    ```
+    ./mi api show
+    ```
+4. Send a POST request using a CURL command or sample client.
+  ```
+curl -v POST -d '{
+    "secretAccessKey":"xxxx",
+    "accessKeyId":"xxxx",
+    "region":"us-east-2",
+    "blocking":"false",
+    "s3Bucket":"eiconnectortest",
+    "s3Key":"addingNumbers.zip",
+    "s3ObjectVersion":"null",
+    "functionName":"eiLambdaConnector",
+    "handler":"addingNumbers.addingNumbers",
+    "role":"arn:aws:iam::610968236798:role/EIConnectorTestRole",
+    "runtime":"python3.7",
+    "apiVersionCreateFunction":"2015-03-31"
+}
+' "http://localhost:8290/createFunction" -H "Content-Type:application/json"  
+ ```
+5. See the following message content.
+ ```
```

Comment:
Fix formatting. This needs to be tabbed.

Reply:
Done.